evacuees/src/API/EMBC.Registrants.API/RegistrationsModule/API.cs

```diff
@@ -176,7 +178,7 @@ public class AnonymousRegistration
     public class RegistrantEvacuation
     {
         [Required]
-        public string ContactId { get; set; }
+        public string BCServicesCardId { get; set; }
```

Comment:
Can this be renamed to `Id`? The best would be to avoid leaking implementation details at the DTOs and endpoints